### PR TITLE
Update LocalStorageCapacityIsolation GA

### PR DESCRIPTION
**Pull request by jingxu97**
update feature GA

### CODE REVIEW

**Comment by xing-yang** [comment]
The content change looks good to me. Not sure why CI failed.

**Comment by xing-yang** [comment]
From the CI:
ERROR 2022/08/15 20:18:31 "GA" is not a valid feature-state, use one of "alpha, beta, deprecated, stable"

File: content/en/docs/concepts/configuration/manage-resources-containers.md [hunk 1]
```diff
@@ -236,7 +236,7 @@ directly or from your monitoring tools.
 ## Local ephemeral storage
 
 <!-- feature gate LocalStorageCapacityIsolation -->
-{{< feature-state for_k8s_version="v1.10" state="beta" >}}
+{{< feature-state for_k8s_version="v1.25" state="GA" >}}
```
Comment (xing-yang) [inline content/en/docs/concepts/configuration/manage-resources-containers.md:239]: GA -> stable

Comment (jingxu97) [reply content/en/docs/concepts/configuration/manage-resources-containers.md]: fixed, thanks!

**Comment by atiratree** [comment]
Hi, it seems that Local ephemeral storage still incorrectly mentions that it is a beta feature. Posted a fix in https://github.com/kubernetes/website/pull/40730

**Pull request merged**